llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -4300,6 +4300,17 @@ getVScaleForTuning(const Loop *L, const TargetTransformInfo &TTI) {
   return TTI.getVScaleForTuning();
 }
 
+static unsigned getEstimatedRuntimeVF(const Loop *L,
```

Comment:
Would be gerat if you could add a comment here to document exactly what we mean by estimated runtime VF here